pyomo/contrib/doe/fim_doe.py

```diff
@@ -623,6 +641,9 @@ def compute_FIM(self, design_values, mode='sequential_finite', FIM_store_name=No
             # call generator function to get scenario dictionary
             scena_gen = Scenario_generator(self.param_init, formula=self.formula, step=self.step)
             scena_gen.generate_sequential_para()
+            scena_set = scena_gen.simultaneous_scenario()
+            self.scena_set = scena_set
+            print(self.scena_set)
```

Comment:
You should have an integer in self.print_level that toggles on print statements such as 0 = no extra print statements, 1 = a few extra print statements, 2 = many extra print statements. For any extra print statements, you need print out extra text to the screen, not just a variable.